File: authandshare/src/main/java/com/fox/authandshare/share/ShareListener.java
package com.fox.authandshare.share;


import com.fox.authandshare.platform.PlatForm;

/**
 * Created by fox.hu on 2018/8/31.
 */

public interface ShareListener {
    void onStart(PlatForm platForm);

    void onComplete(Object response);

    void onError(String errorMsg);

    void onCancel();
}

File: authandshare/src/main/java/com/fox/authandshare/share/ShareListener.kt
package com.fox.authandshare.share


import com.fox.authandshare.platform.PlatForm

/**
 * Created by fox.hu on 2018/8/31.
 */

interface ShareListener {
    fun onStart(platForm: PlatForm)

    fun onComplete(response: Any)

    fun onError(errorMsg: String)

    fun onCancel()
}
